fedimintd/src/distributed_gen.rs

```diff
@@ -189,14 +189,21 @@ impl DistributedGen {
                 finality_delay,
                 password,
             } => {
+                let mut module_gens_params = ServerModuleGenParamsRegistry::default();
+                attach_default_module_gen_params(
+                    &mut module_gens_params,
+                    max_denomination,
+                    network,
+                    finality_delay,
+                );
```

Comment:
We probably want to get rid of this special casing eventually?

Reply:
Agreed.